src/app/app.component.ts

```diff
@@ -1,13 +1,13 @@
-import {Component} from '@angular/core';
-import {AuthenticationService} from './services/authentication.service';
+import {Component} from "@angular/core";
+import {AuthenticationService} from "./services/authentication.service";
 
 @Component({
-  selector: 'app-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  selector: "app-root",
+  templateUrl: "./app.component.html",
+  styleUrls: ["./app.component.scss"],
 })
 export class AppComponent {
-  title = 'Inetum-RealJobs'
+  title = "Inetum-RealJobs";
 
   constructor(private authService: AuthenticationService) {
```

Comment:
private are indicated in JS mostly with underscore at the beginning 
private _authService AuthenticationService

Reply:
I don't know where you got this from, but I've never heard of that before as a normal thing

It's for when you need getters and settings, but you don't need them for dependency injection